### PR TITLE
website: add azure storage options for enterprise

### PR DESCRIPTION
**Pull request by pearkes**
This documents the additional backup target for the snapshot agent.

Originally written by @petems.

### CODE REVIEW

File: website/source/docs/commands/snapshot/agent.html.markdown.erb [hunk 1]
```diff
@@ -256,6 +261,18 @@ The following example IAM policy document assumes that the `aws-s3-bucket` is `c
 }
 ```
 
+#### Azure Blob Storage options
+
+From Consul Enterprise version `1.4.X+` onwards, you can store snapshots in Azure Blob storage.
+
+* `azure-blob-account_name` and `-azure-blob-account_key` - These arguments supply
```
Comment (freddygv) [inline website/source/docs/commands/snapshot/agent.html.markdown.erb]:
```suggestion
* `-azure-blob-account-name` and `-azure-blob-account-key` - These arguments supply
```

File: website/source/docs/commands/snapshot/agent.html.markdown.erb [hunk 2]
```diff
@@ -256,6 +261,18 @@ The following example IAM policy document assumes that the `aws-s3-bucket` is `c
...
+  authentication information for connecting to Azure Blob storage.
+
+* `azure-blob-container_name` - Container to use. Required for Azure blob storage, and setting this
```
Comment (freddygv) [inline website/source/docs/commands/snapshot/agent.html.markdown.erb]:
```suggestion
* `-azure-blob-container-name` - Container to use. Required for Azure blob storage, and setting this
```

File: website/source/docs/commands/snapshot/agent.html.markdown.erb [hunk 3]
```diff
@@ -256,6 +261,18 @@ The following example IAM policy document assumes that the `aws-s3-bucket` is `c
...
+  disables local storage.
+
+** Note: This currently only works on non-Solaris platforms due to library limitations **
```
Comment (freddygv) [inline website/source/docs/commands/snapshot/agent.html.markdown.erb]: Should this line come before the flags? If it's worth the disclaimer maybe it should be at/near the top.

Comment (petems) [reply website/source/docs/commands/snapshot/agent.html.markdown.erb]: Fixed! 😄

**Comment by petems** [comment]
Fixed and rebased according to @freddygv's changes

File: website/source/docs/commands/snapshot/agent.html.markdown.erb (hunk 3 above)
Comment (pearkes) [reply website/source/docs/commands/snapshot/agent.html.markdown.erb]: Thanks both!

**Pull request merged**